commit 56777489abcb26d75c2b7ddba53d1c967bc802c9 (refs/changes/43/1443/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-02-06T14:24:54-08:00 (2 years, 8 months ago)
    
    MANTA-3142 electric-moray A record ttls could be higher
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/sapi_manifests/registrar/template.in b/sapi_manifests/registrar/template.in
index ecc66a0..5b86e90 100644
--- a/sapi_manifests/registrar/template.in
+++ b/sapi_manifests/registrar/template.in
@@ -11,7 +11,7 @@
       },
       "ttl": 60
     },
-    "ttl": 30,
+    "ttl": 60,
     "ports": [@@PORTS@@]
   },
 
